assembl/static2/js/app/components/debate/thread/threadView.jsx

```diff
@@ -50,9 +57,10 @@ class ThreadView extends React.Component<Props> {
         ) : null}
         <Grid fluid className="background-grey">
           <div id="thread-view" className="max-container background-grey">
-            <PostsFilterMenu />
+            <ThreadPostsFilterMenu stickyOffset={60} stickyTopPosition={200} />
```

Comment:
The 60 and 200 are fixed values? This ok on responsive screens

Reply:
They are, i first set them in css, but this was not very maintainable. This allows to change it easily when we add a new filter menu or when we move it. This allows us to compute it when necessary.

Reply:
I have tested it on responsive screen, it is ok (it is about height, not width)